src/emqx_ws_connection.erl

```diff
@@ -257,11 +257,15 @@ websocket_init([Req, Opts]) ->
         andalso maps:get(proxy_header, Req) of
             #{src_address := SrcAddr, src_port := SrcPort, ssl := SSL} ->
                 SourceName = {SrcAddr, SrcPort},
-                %% Notice: Only CN is available in Proxy Protocol V2 additional info
+                %% Notice: CN is only available in Proxy Protocol V2 additional info.
+                %% `CN` is unsupported in Proxy Protocol V1
+                %% `pp2_ssl_cn` is required by config `peer_cert_as_username` or `peer_cert_as_clientid`.
+                %% It will be parsed by esockd.
+                %% See also `emqx_channel:setting_peercert_infos/3` and `esockd_peercert:common_name/1`
                 SourceSSL = case maps:get(cn, SSL, undefined) of
-                             undeined -> nossl;
-                             CN -> [{pp2_ssl_cn, CN}]
-                           end,
+                                undefined -> undefined;
```

Comment:
[esockd_peercert.erl#L34-L40](https://github.com/emqx/esockd/blob/5.8.7/src/esockd_peercert.erl#L34-L40)
```erlang
-spec(common_name(nossl | undefined | peercert()) -> undefined | binary()).
common_name(nossl)     -> undefined;
common_name(undefined) -> undefined;
common_name(Cert) when is_binary(Cert) ->
    esockd_ssl:peer_cert_common_name(Cert);
common_name(PP2Info) when is_list(PP2Info) ->
    proplists:get_value(pp2_ssl_cn, PP2Info).
```
in case the `nossl`, `undefined`, `[{pp2_ssl_cn, undefined}]` all returns `undefined`.
So the typo fix won't effects any feature.